### PR TITLE
Created register page

### PR DESCRIPTION
**Pull request by Afterburn7**

### CODE REVIEW

File: Client/Client/RegisterPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+using System.Text.RegularExpressions;
+
+namespace Client;
+
+public partial class RegisterPage : ContentPage
+{
+	public RegisterPage(RegisterViewModel vm)
+	{
+		InitializeComponent();
+		BindingContext = vm;
```
Comment (Nedux) [inline Client/Client/RegisterPage.xaml.cs:11]: Add line "Shell.SetTabBarIsVisible(this, false);"  To remove navigation pane.

File: Client/Client/RegisterPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	}
+
+    bool isValidEmailRegex(string email)
+    {
+        /*bool isEmail = Regex.IsMatch(email,
```
Comment (Nedux) [inline Client/Client/RegisterPage.xaml.cs:16]: Remove unused comment

File: Client/Client/ViewModels/RegisterViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
```
Comment (Nedux) [inline Client/Client/ViewModels/RegisterViewModel.cs:1]: Delete unused component

File: Client/Client/RegisterPage.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+        if (isValidEmailRegex(Username.Text) == false)
+        {
+           
+            await DisplayAlert("Error", "Your email adress is incorrect!", "OK");
+            Username.Text = string.Empty;
+            Password.Text = string.Empty;
+            RepeatedPassword.Text = string.Empty;
+
+        } else
```
Comment (Nedux) [inline Client/Client/RegisterPage.xaml.cs:44]: We did not agree on syntax rules, but probably new line here should look better:
...
}
else 
...

**Review by Nedux — COMMENTED** [review]
Problems that should be fixed
Also don't forget to use the project's board and register tickets:
https://github.com/users/Nedux/projects/1

P.S. If you will have some time it would be cool if you would add regex for password validation. (added new ticket to mentioned board).

File: Client/Client/RegisterPage.xaml.cs (hunk 1 above)
Comment (Afterburn7) [reply Client/Client/RegisterPage.xaml.cs]: Solved

File: Client/Client/RegisterPage.xaml.cs (hunk 2 above)
Comment (Afterburn7) [reply Client/Client/RegisterPage.xaml.cs]: Solved

File: Client/Client/RegisterPage.xaml.cs (hunk 4 above)
Comment (Afterburn7) [reply Client/Client/RegisterPage.xaml.cs]: Solved

**Pull request merged**